WordPress/Resources/de.lproj/Localizable.strings

```diff
@@ -7462,7 +7462,7 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 "Tap [bookmark-outline] to save a post to your list." = "Tippe auf [bookmark-outline], um einen Beitrag in deiner Liste zu speichern.";
 
 /* Subtitle instructing the user to tap the dismiss button to leave the log in flow */
-"Tap dismiss and head back to your web browser to continue." = "Tippe auf „Dismiss“ (Verwerfen) und navigiere zu deinem Webbrowser zurück, um fortzufahren.";
+"Tap dismiss and head back to your web browser to continue." = "Tippe auf Überspringen und navigiere zu deinem Webbrowser zurück, um fortzufahren.";
```

Comment:
This is the fix for the issue @tiagomar picked up in the QR code login:

<img width="460" alt="image" src="https://user-images.githubusercontent.com/1218433/177968173-f4489223-db8f-4bc4-ab4f-8acacc7b5430.png">

Reply:
In the long run we should probably ask developers to make those two strings be linked to one another, i.e. the prompt to include a `%s` in place where the button title should be in the copy, and insert that dynamically. Have Code Wranglers been notified about this (GitHub issue or similar) so that someone can work on fixing this at some point?